### ISSUE TITLE
FXIOS-518 ⁃ Add Glean first_session ping with Adjust properties to Firefox iOS telemetry

### ISSUE THREAD

**Issue by magorlick** [mozilla-mobile/firefox-ios#6800]
### Why/User Benefit/User Problem
The goal of including this ping is to help us do acquisition channel attribution for retention metrics. We needed this ping to have an independent schedule to help ensure we weren't missing people that bounce or cases where the properties weren't returned in time by Adjust to make the ping schedule for the metrics ping.  This is critical in finding ways to grow the mobile business and is currently not very robust on iOS. 

Context:
Retention using Glean is derived from examining when people come back to the product using the Baseline ping.  However we know we are missing the first session attribution for some users and integrity of this information is particularly low for iOS. 

### Acceptance Criteria (how do I know when I’m done?)
Acquisition is complex and more than the campaign to determine acquisition channel attribution that links specific tactics to downstream outcomes. We need all of these properties included in the first_session custom event.

**- I can see a first_session custom ping in Glean
- The ping includes an Adjust response for network, adgroup, creative in addition to campaign
- The ping should include standard client_info including the client_id and other properties.**

Related issues on Fenix. Please make iOS in parity with these requirements.  Also includes threads on common issues during implementation.  Sawyer on the Fenix team is a great resource for questions.
https://github.com/mozilla-mobile/fenix/issues/7295
https://github.com/mozilla-mobile/fenix/issues/11392

### Not included
Device-level identifiers are out of scope.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-518)

**Comment by athomasmoz** [comment]
Blocked on decision from #6882

**Comment by athomasmoz** [comment]
Closing this ticket as we have decided to remove Adjust #6882
